parallel-consumer-python/pyallel_consumer/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.confluent.parallelconsumer</groupId>
+    <artifactId>python</artifactId>
+    <version>0.1.0a0</version>
+    <properties>
+        <jackson.version>2.13.2</jackson.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>io.confluent.parallelconsumer</groupId>
+            <artifactId>parallel-consumer-core</artifactId>
+            <version>0.5.2.3</version>
```

Comment:
even though it's not reused yet - lets extract this as a property as it's so critical..
in fact, let's use the parent' id - it'll get release as non snapshot when the rest of PC is released

Reply:
so it will be managed by maven rep management

Reply:
I have it pulling from the project root pom now
But means the python package fails as when it executes `./mvnw install` it fails to find `0.5.2.4-SNAPSHOT`

Reply:
Have fixed this
The parallel-consumer-core jar file is now packaged inside the python package and installed locally on startup on the client
This means the python package can now be built and used with SNAPSHOT versions of the java library, rather than just versions available on a public maven repo